caffe2/contrib/mpscnn-fb/mpscnn_test.mm

```diff
@@ -655,7 +656,7 @@ void testMPSCNN() {
     }
   }
 
-  {
+  /*{
```

Comment:
IMO it's a bit cleaner to have just `if (CPUOperatorRegistry()->Has(<optype>) { ...}`, but it's on my list to refactor all of this so not a big deal.